app/_scss/_theme_vars.scss

```diff
@@ -61,7 +61,7 @@ $Theme-font-sizes: (
         line-height: 1.86 // 26px / 14px
       ),
       large: (
-        font-size: 20px,
+        font-size: 18px,
```

Comment:
I did this with @johnny1337, it seems that 20px wasn't the right size attending at the rest of the proportion.